src/routes/tools/extreme-precipitation/_ExploreData.svelte

```diff
@@ -0,0 +1,268 @@
+<script>
+  import { Loading } from "carbon-components-svelte";
+  import { format } from "d3-format";
+
+  // Helpers
+  import { SMALL_SCALE_BOUNDARIES } from "../_common/constants";
+  import {
+    flattenData,
+    groupDataByYear,
+    groupDataByDay,
+    formatDataForExport,
+  } from "../_common/helpers";
+
+  // Components
+  import { Dashboard } from "~/components/tools/Partials";
+  import SettingsPanel from "./_SettingsPanel.svelte";
+  import StatsPanel from "./_StatsPanel.svelte";
+  import ExtremeHeatChart from "./_ExtremeHeatChart.svelte";
+  import ChartTitle from "./_ChartTitle.svelte";
+
+  // Store
+  import {
+    scenarioStore,
+    locationStore,
+    modelsStore,
+    datasetStore,
+    isFetchingStore,
+  } from "../_common/stores";
+  import {
+    climvarStore,
+    indicatorStore,
+    thresholdStore,
+    thresholdTypeStore,
+    durationStore,
+    dataStore,
+    returnPeriodStore,
+  } from "./_store";
+
+  const { location, boundary } = locationStore;
+  const { climvar } = climvarStore;
+  const { scenario } = scenarioStore;
+  const { indicator } = indicatorStore;
+  const { titles } = datasetStore;
+  const { intensity, events, frequency, duration } = dataStore;
+
+  let data;
+  let dataByDate;
+  let showDownload = false;
+  let showShare = false;
+  let showChangeLocation = false;
+  let showLearnMore = false;
+
+  let ChangeLocation;
+  let DownloadChart;
+  let ShareLink;
+  let LearnMoreModal;
+
+  let bookmark;
+  let learnMoreProps = {};
+  let metadata;
+  let csvData;
+  let printContainer;
+  let printSkipElements;
+
+  $: chartDescription = $indicator.description;
+
+  $: chartTitle = $location.title;
```

Comment:
@mukhtyar I noticed the location isn't updating in the title, you'll probably need to use `svelte.afterUpdate`